app/src/controllers/status.js

```diff
@@ -0,0 +1,29 @@
+const errorToProblem = require('../components/errorToProblem');
+const { objectQueueService } = require('../services');
+
+// TODO: Reconsider this value if we add more status endpoints from different services
+const SERVICE = 'ObjectQueueService';
+
+/**
+ * The Status Controller
+ */
+const controller = {
+  /**
+   * @function syncStatus
+   * Lists all known identity providers
```

Comment:
Not sure if I've missed anything here,  but does IDP have anything to do with the object queue?

Reply:
Whoops that was a copypasta mistake. Updated.